src/style.css

```diff
@@ -1,3 +1,72 @@
+* {
+  margin: 0;
+}
+
+html {
+  font-family: Roboto, serif;
+  font-size: 12px;
+
+  --logo-size: 40px;
+  --link-height: 60px;
+  --link-color: #00acdc;
```

Comment:
Ensure that the CSS variables (custom properties) are supported in the browsers you are targeting. If you need to support older browsers, consider providing fallback values.